src/test/java/uk/gov/hmcts/divorce/common/notification/ResendConditionalOrderPronouncedNotificationTest.java

```diff
@@ -0,0 +1,49 @@
+package uk.gov.hmcts.divorce.common.notification;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.hmcts.divorce.systemupdate.service.print.ConditionalOrderPronouncedPrinter;
+
+import static org.mockito.Mockito.verify;
+import static uk.gov.hmcts.ccd.sdk.type.YesOrNo.NO;
+import static uk.gov.hmcts.ccd.sdk.type.YesOrNo.YES;
+import static uk.gov.hmcts.divorce.document.model.DocumentType.CONDITIONAL_ORDER_GRANTED_COVERSHEET_APP_1;
+import static uk.gov.hmcts.divorce.document.model.DocumentType.CONDITIONAL_ORDER_GRANTED_COVERSHEET_APP_2;
+import static uk.gov.hmcts.divorce.testutil.TestDataHelper.validJointApplicant1CaseData;
+
+@ExtendWith(MockitoExtension.class)
+public class ResendConditionalOrderPronouncedNotificationTest {
+
+    @Mock
+    private ConditionalOrderPronouncedPrinter conditionalOrderPronouncedPrinter;
+
+    @InjectMocks
+    private ResendConditionalOrderPronouncedNotification underTest;
+
+    @Test
+    public void shouldSendLetterToOfflineApplicant1WhenCoPronouncedCoverLetterRegenerated() {
```

Comment:
Could also test that nothing is sent if CoPronouncedCoverLetterRegenerated is No